sirepo/srunit.py

```diff
@@ -6,11 +6,14 @@
 """
 from __future__ import absolute_import, division, print_function
 from pykern import pkcompat
+from pykern import pkjson
 from pykern.pkcollections import PKDict
+from sirepo.pkcli import service
```

Comment:
Move this down where it is needed. It's especially important here, because service brings in other modules.

Reply:
https://github.com/radiasoft/sirepo/commit/66da4701448010669a6cbfea717df20863acec62